modules/shimmer/1.0/etc/merge_vcfs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Could you remove this script? I see that the per-chromosome idea failed (rats!) but you shouldn't merge vcf files like this anyway. You should always use `bcftools concat` followed by `bcftools sort` and `bcftools index`, or else downstream tools will complain/break.

Reply:
What was the problem with running this per-chromosome?